bci-game/Assets/HealAnim.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealAnim : MonoBehaviour
+{
+    private float y;
+    // Start is called before the first frame update
+    void Start()
+    {
+        y = BaseY();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        Vector3 currentPos = transform.position;
+        currentPos.y = Mathf.PingPong(Time.time * 0.5f, 0.3f) + y;
+        transform.position = currentPos;
+    }
+
+    float BaseY()
```

Comment:
```suggestion
    private float BaseY()
```

Along with the other comment on colliders, I'll be honest I'm a bit unsure why this class is called HealAnim?